salt/states/zabbix_host.py

```diff
@@ -224,17 +250,53 @@ def _interface_format(interfaces_data):
         for k, v in inv_item.items():
             new_inventory[k] = str(v)
 
+    if "visible_name" not in kwargs:
+        visible_name = None
+    else:
+        visible_name = kwargs.pop("visible_name")
+
+    if kwargs:
+        host_properties_definition = [
+            "description",
+            "inventory_mode",
+            "ipmi_authtype",
+            "ipmi_password",
+            "ipmi_privilege",
+            "ipmi_username",
+            "status",
+            "tls_connect",
+            "tls_accept",
+            "tls_issuer",
+            "tls_subject",
+            "tls_psk_identity",
+            "tls_psk",
+        ]
```

Comment:
IMO this list should be unrestricted. Everything remaining in kwargs should be passed in the host definition. This allows using new Zabbix properties without having to wait for an update to Salt.

Reply:
At first I thought the same and just forwarded the remaining kwargs. But while coding I changed my mind for two reasons:

1. Some host properties needs to be adjusted before forwarding, like the existent "proxy_host" and "inventory". I guess it's better to have a new non-mandatory option ignored than sent wrongly.
2. The host properties set is very stable. From Zabbix 3.0 (2016) to 5.0 (2020) only the "tags" property was added. And this one is just the kind of property that I described before.

I still don't know the best way to handle the "tags" object, so I leave it out of this allowed list, until I figure out a good way. But all the other writable properties are supported and I can create and update a host on Zabbix 5.0.

Now I am writing the tests and fixing some bugs that the tests found, like failing when a host didn't have any additional property.